src/main/kotlin/no/nav/syfo/domain/AktivitetskravStatus.kt

```diff
@@ -11,7 +11,7 @@ enum class AktivitetskravStatus(val isFinal: Boolean) {
     AUTOMATISK_OPPFYLT(true),
     FORHANDSVARSEL(false),
     STANS(true),
-    IKKE_OPPFYLT(true),
+    IKKE_OPPFYLT(true), // TODO: Fjerne / deprecate IKKE_OPPFYLT?
```

Comment:
Hva er forskjellen på `IKKE_OPPFYLT` og `STANS`? 🤔

Reply:
Husker ikke helt nå, men det er `IKKE_OPPFYLT` vi bruker. @andersrognstad husker kanskje?

Reply:
Godt spørsmål, usikker på hva som var tenkt her :thinking: I dag settes vel bare `IKKE_OPPFYLT` og innstillingen skrives i Gosys. Kan jo hende vi vil fortsette å bruke `IKKE_OPPFYLT`, men at funksjonaliteten utvides med mulighet for å skrive innstillingen i Modia. Hvis vi skal endre til å bruke `STANS` når vi har innstilling på plass (og slutte med `IKKE_OPPFYLT`) så har det jo også betydning for datavarehus/statistikk.

Reply:
Hvis vi tenker `STANS`/`INNSTILLING_OM_STANS` er et tydeligere navn så kan vi jo si ifra til dvh 😄 Er jo litt kjedelig med utdaterte navn fordi statistikk har begynt å telle det gamle